sktime/registry/_lookup.py

```diff
@@ -90,6 +97,11 @@ def all_estimators(
             in alphabetical order of class name, where
             ``name`` is the estimator class name as string
             ``class`` is the actual class
+        if return_tags are passed:
```

Comment:
It looks like this docstring was incomplete!
Before you worked on it already.

Would you mind fixing this, while you´re on it?
The "returns"  docstring needs to say what the output is, depending on `return_names` and `as_dataframe`. Might be a little fiddly, apologies, but once that´s done it´s more natural to integrate the effect of `return_tags`.

Currently it´s referring to "columns" which have no clear reference frame, because there is no dataframe that has previously been mentioned (it appears only in the case `as_dataframe=True` which is not documented!)

Reply:
Hey!  I just updated the docstring.  I hope both the explanation of the new functionality, as well as the fleshed out returns section of the docstring makes it easier for users to understand what is going on.